docker/Dockerfile

```diff
@@ -112,7 +122,55 @@ COPY shared/ /tmp/shared
 RUN rustup update nightly && \
     rustup override set nightly && \
     cargo install --locked cargo-pgrx --version 0.11.2 --force && \
-    cargo pgrx package --features simd --pg-config "/usr/lib/postgresql/${PG_VERSION_MAJOR}/bin/pg_config"
+    cargo pgrx init "--pg${PG_VERSION_MAJOR}=/usr/lib/postgresql/${PG_VERSION_MAJOR}/bin/pg_config"
+
+# Use the build argument to update the version in Cargo.toml
+RUN cargo pgrx package --features simd --pg-config "/usr/lib/postgresql/${PG_VERSION_MAJOR}/bin/pg_config"
+
+
+RUN checkinstall --default -D --nodoc --install=no --fstrans=no --backup=no --pakdir=/tmp \
+    --exclude=/root/.cargo \
+    --pkgname $(cargo get package.name) \
+    --pkgversion $(cargo get package.version) \
+    --maintainer "ParadeDB <support@paradedb.com>" \
+    -- cargo pgrx install --release --features simd --pg-config "/usr/lib/postgresql/${PG_VERSION_MAJOR}/bin/pg_config"
+
+######################
+# pg_sparse
+######################
+FROM builder as builder-pg_sparse
+
+WORKDIR /tmp/pg_sparse
+COPY pg_sparse/ /tmp/pg_sparse
+
+RUN make clean && \
+    make USE_PGXS=1 OPTFLAGS="" -j && \
+    checkinstall --default -D --nodoc --install=no --fstrans=no --backup=no --pakdir=/tmp \
+    --pkgname pg-sparse \
+    --pkgversion $(awk -F "= '|\''" '/default_version/ {print $2}' svector.control) \
+    --maintainer "ParadeDB <support@paradedb.com>" \
+    -- make USE_PGXS=1 install -j
+
+######################
+# pgvector
+######################
```

Comment:
I think this whole pgvector block can be removed and we can just use the `apt` one :)

Reply:
done! I switched this to a generic "3rd party extensions" block but still pulling from the apt repo directly